cli/command/system/inspect.go

```diff
@@ -103,38 +102,53 @@ func inspectPlugin(ctx context.Context, dockerCli *command.DockerCli) inspect.Ge
 
 func inspectAll(ctx context.Context, dockerCli *command.DockerCli, getSize bool, typeConstraint string) inspect.GetRefFunc {
 	var inspectAutodetect = []struct {
-		ObjectType      string
-		IsSizeSupported bool
-		ObjectInspector func(string) (interface{}, []byte, error)
+		objectType      string
+		isSizeSupported bool
+		isSwarmObject   bool
+		objectInspector func(string) (interface{}, []byte, error)
 	}{
-		{"container", true, inspectContainers(ctx, dockerCli, getSize)},
-		{"image", false, inspectImages(ctx, dockerCli)},
-		{"network", false, inspectNetwork(ctx, dockerCli)},
-		{"volume", false, inspectVolume(ctx, dockerCli)},
-		{"service", false, inspectService(ctx, dockerCli)},
-		{"task", false, inspectTasks(ctx, dockerCli)},
-		{"node", false, inspectNode(ctx, dockerCli)},
-		{"plugin", false, inspectPlugin(ctx, dockerCli)},
+		{"container", true, false, inspectContainers(ctx, dockerCli, getSize)},
+		{"image", false, false, inspectImages(ctx, dockerCli)},
+		{"network", false, false, inspectNetwork(ctx, dockerCli)},
+		{"volume", false, false, inspectVolume(ctx, dockerCli)},
+		{"service", false, true, inspectService(ctx, dockerCli)},
+		{"task", false, true, inspectTasks(ctx, dockerCli)},
+		{"node", false, true, inspectNode(ctx, dockerCli)},
+		{"plugin", false, false, inspectPlugin(ctx, dockerCli)},
```

Comment:
Cosmetic: this may be easier to read if written as follows:

```
{
        objectType: "container",
        isSizeSupported: true,
        objectInspector: inspectContainers(ctx, dockerCli, getSize),
},
{
        objectType: "image",
        objectInspector: inspectImages(ctx, dockerCli),
},
...
```

This would cut down on some noise. Only fields that differ from the default value need to be specified.

The existing code is fine and I don't want to slow down this fix, but thought I would offer the suggestion.